doc/tutorials/nvmx_virtualization.rst

```diff
@@ -0,0 +1,289 @@
+.. _nested_virt:
+
+Enable Nested Virtualization
+############################
+
+With nested virtualization enabled, it's possible to run virtual machine
+instances inside of level 1 guests. Though it seems both nested guests and
+User VMs are launched from the guest VM, the distinction between them is
+that, in the case of User VM, hypervisor is not running on Service OS, and
+besides the device model, ACRN provides all the support to run the User VMs.
+While for nested guests, an instance of guest hypervisor is running on the
+guest to manage the nested VMs.
+
+The conventional single-level virtualization has two levels - the host
+(ACRN hypervisor) and the guest VMs. With nested virtualization enabled,
+ACRN is able to run guests with their associated virtual machines, which
+has three levels:
+
+* The host (ACRN hypervisor), which we call L0 hypervisor, or L0 VMM
+* The guest hypervisor, which we call L1 hypervisor, or L1 VMM
+* The nested guest, which we call L2 guests
+
+.. figure:: images/nested_1.png
+   :width: 700px
+   :align: center
+
+   Generic Nested Virtualization
+
+
+High Level ACRN Nested Virtualization Design
+********************************************
+
+The high-level design of nested virtualization is shown in :numref:`nested_virt_hld`.
+ACRN. Nested VMX is enabled by so by allowing a guest to use VMX
+instructions, and emulating them using the single level of VMX available
+in the hardware.
+
+A logical processor uses virtual-machine control data structures (VMCSs) while
+it is in VMX operation. The core of the NVMX emulation is ACRN builds VMCS02
```

Comment:
What does NVMX stand for, compared to VMX? Is that the abbreviation for Nested VMX?